app/Http/Requests/UpdateUserRequest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdateUserRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
```

Comment:
return $this->user()->can('update', $this->route('user'));
OR try
return $this->authorize('update', $this->route('user'));